code/modules/mob/living/autohiss.dm

```diff
@@ -0,0 +1,110 @@
+
+#define AUTOHISS_OFF 0
+#define AUTOHISS_BASIC 1
+#define AUTOHISS_FULL 2
+
+#define AUTOHISS_NUM 3
+
+
+/mob/living/proc/handle_autohiss(message, datum/language/L)
+	return message // no autohiss at this level
+
+/mob/living/carbon/human/handle_autohiss(message, datum/language/L)
+	if(!client || client.autohiss_mode == AUTOHISS_OFF) // no need to process if there's no client or they have autohiss off
+		return message
+	return species.handle_autohiss(message, L, client.autohiss_mode)
+
+/client
+	var/autohiss_mode = AUTOHISS_OFF
+
+/client/verb/toggle_autohiss()
+	set name = "Toggle Auto-Accent"
+	set desc = "Toggle automatic accents for your species"
+	set category = "OOC"
```

Comment:
yeah, like I said, I don't really care either; I just found it odd that it was shoved into the OOC tab when things like "character pose" and such are in the IC tab (I think).